EIPS/eip-758.md

```diff
@@ -9,30 +9,32 @@ created: 2017-11-09
 ---
 
 ## Simple Summary
-Provide a way for external callers to access the return data of functions executed during Ethereum transactions.
+Provide a way for external callers to be notified of completed transactions, and access the return data of functions executed when a transaction is mined.
 
 ## Abstract
-When a new transaction is submitted successfully to an Ethereum node, the node responds with the transaction's hash.  If the transaction involved the execution of a contract function that returns data, the data is discarded.  If the return data is state-dependent, which is common, there is no straightforward way for the caller to access or compute the return data.  This EIP proposes that callers should be able to subscribe to (or poll for) the return data of their transactions.  The Ethereum node then sends the return data to the caller when the transactions are sealed.
+When a new transaction is submitted successfully to an Ethereum node, the node responds with the transaction's hash.  If the transaction involved the execution of a contract function that returns data, the data is discarded.  If the return data is state-dependent, which is common, there is no straightforward way for the caller to access or compute the return data.  This EIP proposes that callers should be able to subscribe to (or poll for) completed transaction events.  The Ethereum node sends the return data to the caller when the transactions are sealed.
```

Comment:
Suggest "completed transactions" instead of "completed transaction events", which could be confused with logged events.